static/src/javascripts-legacy/projects/commercial/modules/hosted/onward.js

```diff
@@ -26,7 +26,7 @@ define([
                     return fastdom.write(function () {
                         var i;
                         for (i = 0; i < placeholders.length; i++) {
-                            placeholders[i].innerHTML = json.html;
+                            placeholders[i].innerHTML = placeholders[i].innerHTML + json.html;
```

Comment:
I recommend using this instead:

```
placeholders[i].insertAdjacentHTML('beforeend', json.html)
```

this won't invalidate the whole content of the element and ultimately render faster.